2.0/parser/filing_parser.py

```
@@ -0,0 +1,1799 @@
+        
+        # Search for the SEC header in the full text
+        try:
+            match = re.search(pattern, fulltext_contents, re.DOTALL | re.IGNORECASE) 
+        except Exception as e:
+            logging.error(f'Failed regex search for SEC header. Error: {e}.')
+            return None
+
+        if match:
+            return match.group(0)  # Return the entire match, which includes the tags
+        else:
+            return None  # Return None if no SEC header is found
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)